crates/nu-protocol/src/shell_error.rs

```diff
@@ -1323,6 +1323,21 @@ This is an internal Nushell error, please file an issue https://github.com/nushe
         #[label = "byte index is not a char boundary or is out of bounds of the input"]
         span: Span,
     },
+
+    /// The config directory could not be found
+    #[error("The config directory could not be found")]
+    #[diagnostic(
+        code(nu::shell::config_dir_not_found),
+        help(
+            r#"On Linux, this would be $XDG_CONFIG_HOME or $HOME/.config.
+On MacOS, this would be $HOME/Library/Application Support.
```

Comment:
maybe that space could be confusing?
what about putting all these inside backticks? e.g.
```suggestion
On MacOS, this would be `$HOME/Library/Application Support`.
```